app/services/symphony_client.rb

```diff
@@ -75,7 +75,7 @@ def renew_items(user, checkouts)
         status[:success] << { renewal: checkout, sirsi_response: nil }
       else
         Rails.logger.error("Renewal (#{checkout.item_key}): #{response.body}")
-        status[:error] << { renewal: checkout, sirsi_response: (error_message(response) || '') }
+        status[:error] << { renewal: checkout, sirsi_response: (renewal_error_message(response) || '') }
```

Comment:
I think this change is a nice improvement. I also think this could be moved to the controller to be consistent with the rest of our code, symphony client just being the thing that performs the request and the controller handles what to do with it. SymphonyClientResponse could even be a model maybe. But, this is not that important, just "thinking outloud" really.

Reply:
yes agreed. I think such change for renewals would also be part of refactoring the holds_controller for another PR.